dbux-code/src/codeUtil/treeView/TraceNode.js

```diff
@@ -2,15 +2,10 @@ import { TreeItemCollapsibleState } from 'vscode';
 import Trace from '@dbux/common/src/types/Trace';
 import traceSelection from '@dbux/data/src/traceSelection';
 import { makeTraceLabel, makeTraceLocLabel } from '@dbux/data/src/helpers/makeLabels';
-import UserActionType from '@dbux/data/src/pathways/UserActionType';
 import allApplications from '@dbux/data/src/applications/allApplications';
 import BaseTreeViewNode from './BaseTreeViewNode';
 
 export default class TraceNode extends BaseTreeViewNode {
-  get clickUserActionType() {
```

Comment:
@MiccWan since this is part of `Pathways`: did you make sure whether pathways works as expected when making this change?
Else, we want to avoid touching features that we are not currently working on

Reply:
Added getter of `clickUserActionType` which returns false to `BaseTreeViewNode`. So by default no `UserEvent` will be send on click.